frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -765,10 +766,15 @@ const crdCardRowStateToProps = ({ k8s }, { crdDescs }) => {
 export const CRDCardRow = connect(crdCardRowStateToProps)((props: CRDCardRowProps) => {
   const internalObjects = getInternalObjects(props.csv);
   const crds = props.crdDescs?.filter(({ name }) => !isInternalObject(internalObjects, name));
+  // if the CSV was created within the last 5 minutes (300000 milliseconds)
+  const recentlyCreated = moment(Date.now()).diff(props.csv.metadata.creationTimestamp) < 300000;
```

Comment:
Isn't 30,000ms == 30s? I would use `5 * 60 * 1000`.

I wonder if we should check the date at all since we could have clock drift between the client and server. I think we should a similar message elsewhere in the UI without the date check.

Reply:
https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx#L420

Reply:
@rhamilto Disregard, I missed a `0` :)